src/effects/lv2/LoadLV2.cpp

```diff
@@ -305,8 +305,11 @@ std::unique_ptr<ComponentInterface>
 LV2EffectsModule::LoadPlugin(const PluginPath & path)
 {
    // Acquires a resource for the application.
-   if (auto plug = GetPlugin(path))
-      return std::make_unique<LV2Effect>(plug);
+   if (auto plug = GetPlugin(path)) {
+      auto result = std::make_unique<LV2Effect>(plug);
+      result->InitializePlugin();
```

Comment:
Seems like same sequence of calls (create and then initialize) is used for each plugin type, so `InitializePlugin` may be done during effect construction and removed from the interface.

Reply:
Maybe, but I will leave this as it is.

It's not part of any virtual function interface -- it is only a coincidental sameness in implementations of `LoadPlugin` cross the families.

There is yet the confusion between plugin and instance of plugin to be worked out.  Some effects (not LV2 though) implement RealtimeAddProcessor by creating another object of the effect subclass.  In those cases we do not want to repeat the plugin initialization, but only do instance initialization.  That was the point of the earlier commit that split and renamed the function `SetHost`.